tests/ShortcutTest.php

```diff
@@ -57,176 +59,137 @@ public function testVipsWop()
-            return $value << 2; 
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value << 2;
         });
         $vips = $this->image->lshift(2)->getpoint(0, 0);
 
         $this->assertEquals($vips, $real);
 
-        $real = self::map_numeric($this->pixel, function ($value) {
-            return $value >> 2; 
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value >> 2;
         });
         $vips = $this->image->rshift(2)->getpoint(0, 0);
 
         $this->assertEquals($vips, $real);
-
     }
 
     public function testVipsBool()
     {
-        $real = self::map_numeric($this->pixel, function ($value) {
-            return $value & 2; 
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value & 2;
         });
         $vips = $this->image->andimage(2)->getpoint(0, 0);
 
         $this->assertEquals($vips, $real);
 
-        $real = self::map_numeric($this->pixel, function ($value) {
-            return $value | 2; 
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value | 2;
         });
         $vips = $this->image->orimage(2)->getpoint(0, 0);
 
         $this->assertEquals($vips, $real);
 
-        $real = self::map_numeric($this->pixel, function ($value) {
-            return $value ^ 2; 
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value ^ 2;
         });
         $vips = $this->image->eorimage(2)->getpoint(0, 0);
 
         $this->assertEquals($vips, $real);
-
     }
 
     public function testVipsRelational()
     {
-        $real = self::map_numeric($this->pixel, function ($value) {
-            if ($value > 38) {
-                return 255;
-            }
-            else {
-                return 0;
-            }
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value > 38 ? 255 : 0;
         });
         $vips = $this->image->more(38)->getpoint(0, 0);
         $this->assertEquals($vips, $real);
 
-        $real = self::map_numeric($this->pixel, function ($value) {
-            if ($value >= 38) {
-                return 255;
-            }
-            else {
-                return 0;
-            }
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            return $value >= 38 ? 255 : 0;
         });
-        $vips = $this->image->moreeq(38)->getpoint(0, 0);
+        $vips = $this->image->moreEq(38)->getpoint(0, 0);
         $this->assertEquals($vips, $real);
 
-        $real = self::map_numeric($this->pixel, function ($value) {
-            if ($value > $value) {
-                return 255;
-            }
-            else {
-                return 0;
-            }
+        $real = self::mapNumeric($this->pixel, function ($value) {
+            // FIXME: Left and right operands are identical
+            return $value > $value ? 255 : 0;
```

Comment:
Should we just return `0` here?

Reply:
I agree, perhaps with a comment saying something like "(x > x) is always false".